src/main/java/org/rulez/demokracia/pdengine/ContainingBeats.java

```diff
@@ -54,12 +55,14 @@ default void initialize(List<CastVote> castVotes) {
 		
 		for (CastVote castVote  : castVotes) {
 			List<RankedChoice> preferences = castVote.getPreferences();
-
+			
```

Comment:
I will not block a merge just for space usage until we have a policy on spaces and braces.

Reply:
Fixed :)